importers/importer.py

```diff
@@ -100,21 +100,20 @@ def import_point(points):
 
         point_id = get_id_of("points", point.point_name)
 
-        for (table, id, tags) in [('buildings', building_id, point.building_type),
-                                  ('rooms', room_id, point.room_type),
-                                  ('devices', device_id, point.device_type),
-                                  ('points', point_id, point.point_type)]:
-            if tags is None:  # TODO: This is just a hotfix. I (ATD) think we need to redesign this.
-                continue
+        for (table, id, tags) in [('buildings', building_id, point.get_building_tags()),
+                                  ('rooms', room_id, point.get_room_tags()),
+                                  ('devices', device_id, point.get_device_tags()),
+                                  ('points', point_id, point.get_point_tags())]:
             for tag in tags:
-                tag_id = get_id_of("tags", tag)
-                if tag_id is None:
-                    execute_and_commit("""INSERT INTO tags (name) VALUES (%s)""", (tag,))
+                if tag is not None:
```

Comment:
Wouldn't it be safe to assume that `tags` will be an array of non-None values? Not necessarily saying we need to change this, but just want to confirm my understanding.

Reply:
I could be wrong, but I think right now, the ____type (device_type for example) is always going in the list of tags regardless of whether or not it is None. I guess we could decide to only add tags to the list that are non-None values or check it here. I'm not sure which I prefer.

Reply:
Yup! I agree it doesn't make sense to ever add none to the tags list. Fixed in a new commit, please give a super quick look and approve when you can :)